### DISCUSSION TITLE
After connecting the wallet via qr  Error switching wallet

### DISCUSSION THREAD

**Discussion by EV-Ain** [WalletConnect/walletconnect-monorepo#3927] [Developer Support]
<template>
  <button @click="modal.open()">Open Connect Modal</button>
  <button @click="modal.open({ view: 'Networks' })">Open Network Modal</button>
</template>

<script setup>
  import { createWeb3Modal, defaultWagmiConfig } from "@web3modal/wagmi/vue";
  import { bscTestnet } from "viem/chains";
  import { useWeb3Modal } from "@web3modal/wagmi/vue";
  import { getAccount, watchAccount } from "@wagmi/core";

  const projectId = "c7627b885f0ac6931fc1a2573a703fe7";

  const chains = [bscTestnet];
  const wagmiConfig = defaultWagmiConfig({ chains, projectId });

  createWeb3Modal({ wagmiConfig, projectId, chains, themeMode: "light" });
  const modal = useWeb3Modal();

  const unwatch = watchAccount(account => {
    console.log("🚀  account:", account);
  });
</script>
![Snipaste_2023-11-15_11-33-33](https://github.com/WalletConnect/walletconnect-monorepo/assets/101503420/f567d7b4-28c6-42c1-93cf-eb84549a5fee)

**Comment by EV-Ain** [comment]
![微信图片_20231115173848](https://github.com/WalletConnect/walletconnect-monorepo/assets/101503420/640a0773-5e2f-4bc9-8077-f46c38429850)

**Reply by EV-Ain** [reply]
Full code

**Comment by EV-Ain** [comment]
I see the same problem here;Switching wallets does not take effect after qrcode connects to wallets; https://web3modal.com/

**Reply by glitch-txs** [reply]
Hi, thanks for reporting, could you share steps to reproduce the issue?

**Comment by EV-Ain** [comment]
> Hi, thanks for reporting, could you share steps to reproduce the issue?您好，感谢您的报告，您能分享重现问题的步骤吗？

My mobile phone used qrcode to connect to the wallet, and the official website reported an error after the mobile phone switched the wallet address

**Comment by EV-Ain** [comment]
Error when copying wallet address on Android phone

**Comment by EV-Ain** [comment]
Chrome, after the user links to the wallet via qr code, does not remind the user to call the wallet when paying

**Comment by EV-Ain** [comment]
@glitch-txs

**Reply by glitch-txs** [reply]
hi, the issue is not clear, could you go through https://github.com/orgs/WalletConnect/discussions/3457
and provide a MRE and steps to reproduce? Thanks
